sdks/typescript/src/util/uuid.ts

```diff
@@ -0,0 +1,10 @@
+export function isValidUUID(uuid: string): boolean {
+  return /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i.test(uuid);
```

Comment:
there must be a way to do this with a library 😅

Reply:
good catch, we already have zod as a dep